faasmcli/faasmcli/tasks/run.py

```diff
@@ -1,23 +1,29 @@
+from copy import copy
 from invoke import task
+from os import environ
 
 from faasmcli.util.shell import run_command
 
 
 @task
-def user(ctx, user, repeats=1):
+def user(ctx, user, repeats=1, wamr=False):
```

Comment:
Another slightly unrelated improvement to make it possible to pass a flag to run with WAMR